tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+		}
+
+		add_filter( 'tribe_events_eb_iframe_html', array( $this, 'iframe_height' ), 10, 3 );
+
+		if ( 'en_US' !== $this->get_chosen_eb_locale() ) {
+			add_filter( 'tribe-eventbrite-base_api_url', array( $this, 'api_url' ) );
+
+			add_filter( 'tribe_eb_api_sync_event', array( $this, 'eb_currency_for_update_create' ), 10, 5 );
+		}
+
+		// originally from https://theeventscalendar.com/knowledgebase/use-the-event-cost-field-with-tickets-installed/
+		// cannot use https://github.com/moderntribe/products-extensions/blob/master/tribe-ext-show-cost-field/index.php because that Extension requires Event Tickets Plus active
+		$show_admin_cost_field = tribe_get_option( $this->opts_prefix . 'show_admin_cost_field' );
+		if ( ! empty( $show_admin_cost_field ) ) {
+			add_filter( 'tribe_events_admin_show_cost_field', '__return_true', 100 );
+		}
+
+		// originally from https://theeventscalendar.com/knowledgebase/moving-the-eventbrite-tickets-box/
+		$eb_new_ticket_location = tribe_get_option( $this->opts_prefix . 'ticket_single_location' );
+		$eb_ticket_default_location = 'tribe_events_single_event_after_the_meta';
+		if ( ! empty( $eb_new_ticket_location ) && $eb_ticket_default_location !== $eb_new_ticket_location ) {
+			$display_tickets = array( tribe( 'eventbrite.main' ), 'print_ticket_form' );
+			remove_action( $eb_ticket_default_location, $display_tickets, 9 );
+			add_action( $eb_new_ticket_location, $display_tickets );
+		}
+	}
+
+	public function before_iframe() {
+		$text = tribe_get_option( $this->opts_prefix . 'content_before' );
+
+		$text = sprintf( '<h3 class="tribe_ext_eventbrite_opts">%s</h3>', $text );
+
+		return $text;
+	}
+
+	public function after_iframe() {
+		$text = tribe_get_option( $this->opts_prefix . 'content_after' );
+
+		$text = sprintf( '<span class="tribe_ext_eventbrite_opts">%s</span>', $text );
+
+		return $text;
+	}
+
+	// originally from https://theeventscalendar.com/knowledgebase/changing-the-height-of-the-eventbrite-tickets-iframe/
+	public function iframe_height( $html, $event_id, $post_id ) {
+		// should only be a positive integer, per validation
+		$new_height = intval( tribe_get_option( $this->opts_prefix . 'iframe_px' ) );
+
+		if ( 200 === $new_height ) {
+			return $html;
+		}
+
+		if ( empty( $new_height ) || false === strpos( $html, 'eventbrite-ticket-embed' ) ) {
+			return $html;
+		}
+
+		$replace_this = 'style="height:200px;';
+
+		$replace_with = sprintf( 'style="height: %dpx;', $new_height ); // integers only
+
+		$new_html = str_replace( $replace_this, $replace_with, $html );
+
+		return $new_html;
+	}
+
+//	public function eb_credit_link_to_locale( $html, $event_id, $post_id ) {
+//
+//		$replace_with = sprintf( 'eventbrite%s', $this->get_eb_tld() );
+//
+//		$new_html = str_replace( 'eventbrite.com', $replace_with, $html );
+//
+//		return $new_html;
+//	}
+
+	public function eb_currency_for_update_create( $args, $mode, $eventbrite_id, $post, $params ) {
```

Comment:
For public functions its best to have a docblock. In lieu of documenting each arg and the return, feel free to `@see` the relevant filter.